### ISSUE TITLE
Yola.activate_trial_subscription deprovisions instead of activates

### ISSUE THREAD

**Issue by blaix** [yola/yolapy#24]
```
>>> partner = yola.create_partner(
...   id='WL_TEST_BOOGIE_WOOGIE',
...   name='TEST',
...   parent_partner_id='WL_YOLA',
...   properties={
...     'website': 'example.com',
...     'available_subscription_types': ['wl_basic']
...   })
>>> user = yola.create_user(
...   email='justin+testyourmight@yola.com',
...   name='Justin',
...   surname='Time',
...   partner_id=partner['id'],
...   preferences={})
>>> sub = yola.create_subscription('wl_basic', user['id'], {'trial': True})
>>> sub['status']
u'active'
>>> sub['properties']['trial']
u'True'
>>> yola.activate_trial_subscription(sub['id'])
>>> activated_sub = yola.get_subscription(sub['id'])
>>> activated_sub['status']
u'deprovisioned'
>>> activated_sub['properties']['trial']
u'True'
```

I don't know if the problem is in yolapy or one of the services. When we dig into this, if it's not a yolapy problem, create an issue in the appropriate project.

**Comment by RayeN** [comment]
Works as expected.  We changed remove trial functionality not so long ago to perform subscription upgrade instead of just removing trial flag (we did that because of partner's expectation that activated subscription should start on the date of activation).   

So, right now your `activate_trial_subscription` should create new non-trial sub (and return it) and the old sub should be deprovisioned.

Let me know if have any questions.
